.nova/Configuration.json

```diff
@@ -1,5 +1,16 @@
 {
+  "editor.default_syntax" : "javascript",
   "index.ignored_file_patterns" : [
+    "prettier.novaextension\/config.json",
+    "prettier.novaextension\/configWorkspace.json",
     "prettier.novaextension\/Scripts\/*"
-  ]
+  ],
+  "prettier.default-config.endOfLine" : "lf",
+  "prettier.default-config.printWidth" : 80,
+  "prettier.default-config.semi" : "Disabled",
+  "prettier.default-config.singleQuote" : "Enabled",
+  "prettier.default-config.useTabs" : "Enabled",
```

Comment:
There's already a `.prettierrc` file, so this isn't necessary?

Reply:
Indeed there is the `.prettierrc` file, it is just syntactic sugar. 😄

Reply:
I'd rather keep the configuration in one place then. Otherwise it's just easy to get conflicts or confusion.